### PR TITLE
[Bug] Cycle time calculation fix, Improving performance for Burndown

### PR DESCRIPTION
**Pull request by achaur11**

### CODE REVIEW

File: taigaProject/src/taigaApi/util/metric_service.py [hunk 1]
```diff
@@ -107,17 +108,12 @@ def calc_burndown_day_data(auth_token, milestone, attribute_key):
     }
     
     total_business_value = 0
```
Comment (shahih25) [inline taigaProject/src/taigaApi/util/metric_service.py]: Can you please create a map for this variable? Similar to what you did for cycle time.

Comment (shahih25) [reply taigaProject/src/taigaApi/util/metric_service.py]: `total_business_value = {
       "bv": 0
}`

Comment (achaur11) [reply taigaProject/src/taigaApi/util/metric_service.py]: Done.

**Review by shahih25 — APPROVED** [review]
LGTM

**Pull request merged**